partition/bootloader.go

```diff
@@ -55,6 +60,28 @@ type Bootloader interface {
 
 	// Name returns the bootloader name
 	Name() string
+
+	// configFile returns the name of the config file
+	ConfigFile() string
+}
+
+// InstallBootConfig install the bootloader config from the gadget
+// snap dir into the right place
+func InstallBootConfig(gadgetDir string) error {
+	for _, bl := range []Bootloader{&grub{}, &uboot{}} {
+		fn, err := find(gadgetDir, filepath.Base(bl.ConfigFile()))
```

Comment:
Thanks, I fixed the name. And I agree, we should define a location, I think toplevel of the snap is fine? If you agree I will make it so and update our gadget snaps to follow that convention.